cmd/_integration-tests/transport/handlers/handlers.go

```diff
@@ -20,6 +21,14 @@ type transportpermutationsService struct{}
 
 // GetWithQuery implements Service.
 func (s transportpermutationsService) GetWithQuery(ctx context.Context, in *pb.GetWithQueryRequest) (*pb.GetWithQueryResponse, error) {
+	spew.Dump(ctx)
+	if rurl := ctx.Value("request-url").(string); rurl != "/getwithquery" {
```

Comment:
This is not really a test. It panics inside the handlers rather than asserting on values in the `http_test.go` file.

See test [TestCtxToCtxViaHTTPHeaderClient](https://github.com/adamryman/go-truss/blob/d1d3f0016e9b975fd8e080182792744e6b6d206a/cmd/_integration-tests/transport/http_test.go#L168) for a header test example.